src/components/Navbar/Navbar.js

```diff
@@ -260,7 +260,12 @@ export default function NavBar() {
               <LoginButton
                 variant="contained"
                 disableElevation
-                onClick={() => loginWithRedirect()}
+                  onClick={() => loginWithRedirect({
```

Comment:
This is actually a bugfix since you always want to pass this in so that people go back to the page they were on